easybuild/easyconfigs/g/gdc-client/gdc-client-1.3.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,134 @@
+    }),
+    ('termcolor', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/termcolor'],
+        'checksums': ['1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b'],
+    }),
+    ('sortedcontainers', '1.5.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/sortedcontainers'],
+        'checksums': ['844daced0f20d75c02ce53f373d048ea2e401ad8a7b3a4c43b2aa544b569efb3'],
+    }),
+    ('intervaltree', '2.0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/intervaltree'],
+        'checksums': ['d5c45ce8874ef04303c02b66327793320aa2027cf25c89188f74be69beaa3c5f'],
+    }),
+    ('itsdangerous', '0.24', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/itsdangerous'],
+        'checksums': ['cbb3fcf8d3e33df861709ecaf89d9e6629cff0a217bc2848f1b41cd30d360519'],
+    }),
+    ('MarkupSafe', '1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/M/MarkupSafe'],
+        'checksums': ['a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665'],
+    }),
+    ('Jinja2', '2.10', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2'],
+        'checksums': ['f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4'],
+    }),
+    ('Werkzeug', '0.13', {
+        'source_urls': ['https://pypi.python.org/packages/source/W/Werkzeug'],
+        'checksums': ['6246e5fc98a505824113fb6aca993d45ea284a2bcffdc2c65d0c538e53e4abd3'],
+    }),
+    ('Flask', '0.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/F/Flask'],
+        'checksums': ['4c83829ff83d408b5e1d4995472265411d2c414112298f2eb4b359d9e4563373'],
+    }),
+    ('progressbar', '2.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/progressbar'],
+        'checksums': ['b2d38a729785149e65323381d2e6fca0a5e9615a6d8bcf10bfa8adedfc481254'],
+    }),
+    ('requests', '2.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests'],
+        'checksums': ['7b7735efd3b1e2323dc9fcef060b380d05f5f18bd0f247f5e9e74a628279de66'],
+    }),
+    ('parcel', '20170814', {
+        'source_tmpl': '50d6124a3e3fcd2a234b3373831075390b886a15.tar.gz',
+        'source_urls': ['https://github.com/LabAdvComp/parcel/archive'],
+        'checksums': ['f7d394a34387358e2b4b2e9f0ba8ebf28bc6bc4567627b69765e91cfe29b6795'],
+    }),
+    (name, version, {
+        'modulename': 'gdc_client',
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/NCI-GDC/gdc-client/archive'],
+        'checksums': ['2a847f7498491fe69f86f163b9adc6ba752e740f5f17791da8c17a336a84751f'],
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
```

Comment:
@fizwit This should be removed, since it's totally useless, see also #7094

Reply:
@fizwit Please remove the `full_sanity_check = True`, it hasn't been supported since EasyBuild 2.2.0 (i.e., it doesn't have any effect all)